### PR TITLE
About

### PR DESCRIPTION
**Pull request by SamLee514**
- Fixed Aadi
- Added Baibhav (with placeholders)
- Update app info

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -35,12 +35,12 @@ const IndexPage: React.FC = () => (
...
       <GeneralDescription
-        title="Board Applications Open!"
+        title="Welcome to VandyHacks!"
         text1={
-          `Welcome to VandyHacks! Our 2021/VH VIII Board applications are currently open until 11:59 PM CST on January 20th, 2021.`
+          `Our 2021/VH VIII Board applications have closed, and we are ecstatic about all our new members!`
         }
         text2={
-          `Join us in making Vanderbilt University's computer science community an even more enjoyable, diverse, and welcoming space.`
+          `Click here to learn more about VandyHacks and what we do as an organization. We hope to see your application for VH XI!`
```
Comment (aadibajpai) [inline src/pages/index.tsx]: bro XI is 11, we want VIII for 8 ig unless it's for like internal recruitment for VH IX

File: src/pages/join.tsx [hunk 2]
```diff
@@ -24,9 +24,9 @@ const join: React.FC<{}> = () => (
     <body>
         <PageLinks />
         <BoardApps 
-            title="2021 Board Applications - Open through January 20th, 2021"
+            title="2021 Board Applications now Closed"
             text={
-                "With 7 different committees (Content, Design, Development, Hacker Experience, Marketing, Operations, and Sponsorship), VandyHacks definitely has something for you! We warmly welcome all years and majors; our top priority is to bring together a group filled with genuine enthusiasm and creative energy. You can learn more about VandyHacks and our application process below."
+                "While we have closed our applications for this season, we still welcome you to learn more about our organization. With 7 different committees (Content, Design, Development, Hacker Experience, Marketing, Operations, and Sponsorship), VandyHacks definitely has something for you! We warmly welcome all years and majors; our top priority is to bring together a group filled with genuine enthusiasm and creative energy. You can learn more about VandyHacks and our application process below."
```
Comment (aadibajpai) [inline src/pages/join.tsx]:
```suggestion
                "While we have closed our applications for this season, we still invite you to learn more about our organization. With 7 different committees (Content, Design, Development, Hacker Experience, Marketing, Operations, and Sponsorship), VandyHacks definitely has something for you! We warmly welcome all years and majors; our top priority is to bring together a group filled with genuine enthusiasm and creative energy. You can learn more about VandyHacks and our application process below."
```

File: src/pages/index.tsx [hunk 3]
```diff
@@ -35,12 +35,12 @@ const IndexPage: React.FC = () => (
     <PageLinks />
     <GeneralBodyContainer>
       <GeneralDescription
-        title="Board Applications Open!"
+        title="Welcome to VandyHacks!"
         text1={
-          `Welcome to VandyHacks! Our 2021/VH VIII Board applications are currently open until 11:59 PM CST on January 20th, 2021.`
+          `Our 2021/VH VIII Board applications have closed, and we are ecstatic about all our new members!`
```
Comment (aadibajpai) [inline src/pages/index.tsx]:
```suggestion
          `Our 2021/VH VIII Board applications have closed, and we welcome all our new members!`
```
ecstatic feels too cool here

File: src/pages/index.tsx (hunk 1 above)
Comment (aadibajpai) [reply src/pages/index.tsx]: yeah ig we need IX

File: src/pages/join.tsx [hunk 4]
```diff
@@ -24,9 +24,9 @@ const join: React.FC<{}> = () => (
     <body>
         <PageLinks />
         <BoardApps 
-            title="2021 Board Applications - Open through January 20th, 2021"
+            title="2021 Board Applications now Closed"
```
Comment (aadibajpai) [inline src/pages/join.tsx]: capitalization is awk here I think, maybe just have "2021 Board Applications closed on January 20th, 2021" so ppl still have an approx idea for next year you know

**Review by aadibajpai — COMMENTED** [review]
looks good, some nitpicks I think

File: src/pages/index.tsx (hunk 1 above)
Comment (SamLee514) [reply src/pages/index.tsx]: LOL wait I got IX and XI mixed

File: src/pages/join.tsx (hunk 4 above)
Comment (SamLee514) [reply src/pages/join.tsx]: Fair

File: src/pages/index.tsx (hunk 3 above)
Comment (SamLee514) [reply src/pages/index.tsx]: are we not cool?

Comment (SamLee514) [reply src/pages/index.tsx]: I feel like "we welcome all our new members" sounds a bit off. How about split the difference and be like we're excited to welcome all our new members?

Comment (aadibajpai) [reply src/pages/index.tsx]: okay sounds good let's do that

Comment (aadibajpai) [reply src/pages/index.tsx]: ecstatic just feels out of place-ish, like it draws attention to itself

**Review by aadibajpai — APPROVED** [review]
sex

**Pull request merged**